### PR TITLE
update docs for tiflash pipeline model

### PR DESCRIPTION
**Pull request by SeaRise**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14733
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2265,22 +2265,14 @@ Query OK, 0 rows affected (0.09 sec)
...
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
 - Persists to cluster: Yes
 - Type: Boolean
-- Default value: `OFF`
+- Default value: `ON`
 - This variable controls whether to enable the new [Pipeline Execution Model](/tiflash/tiflash-pipeline-model.md) in TiFlash.
 - When this variable is set to `OFF` to disable the pipeline execution model, the query pushed down to TiFlash will be executed using the original stream model.
 - When this variable is set to `ON` to enable the pipeline execution model, the query pushed down to TiFlash will be executed using the new pipeline execution model.
```
Comment (Oreoxmt) [inline system-variables.md:2274]:
```suggestion
- When this variable is set to `ON` to enable the pipeline execution model, the query pushed down to TiFlash will be executed using the new pipeline execution model.
- When this variable is set to `OFF` to disable the pipeline execution model, the query pushed down to TiFlash will be executed using the original stream model.
```

File: system-variables.md [hunk 2]
```diff
@@ -2265,22 +2265,14 @@ Query OK, 0 rows affected (0.09 sec)
 
 <CustomContent platform="tidb">
```
Comment (Oreoxmt) [inline system-variables.md:2266]:
```suggestion
```

File: system-variables.md [hunk 3]
```diff
@@ -2265,22 +2265,14 @@ Query OK, 0 rows affected (0.09 sec)
...
 
-> **Note:**
->
-> - The pipeline execution model is currently an experimental feature and is not recommended to use in production environments.
-> - The pipeline execution model does not support the following features. When the following features are enabled, even if `tidb_enable_tiflash_pipeline_model` is set to `ON`, the query pushed down to TiFlash will still be executed using the original stream model.
->
->     - [Join operator spill to disk](#tidb_max_bytes_before_tiflash_external_join-new-in-v700)
->     - [TiFlash Disaggregated Storage and Compute Architecture and S3 Support](/tiflash/tiflash-disaggregated-and-s3.md)
-
 </CustomContent>
```
Comment (Oreoxmt) [inline system-variables.md:2277]:
```suggestion
```

File: system-variables.md (hunk 1 above)
Comment (Oreoxmt) [reply system-variables.md:2274]: Please remove L2278-L2296

https://github.com/pingcap/docs/blob/580081a8c166d99972f5666a67939866c11c4928/system-variables.md?plain=1#L2278-L2296

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -2261,45 +2261,31 @@ Query OK, 0 rows affected (0.09 sec)
 
 </CustomContent>
 
-### tidb_enable_tiflash_pipeline_model <span class="version-mark">New in v7.2.0</span>
+### tiflash_enable_pipeline_model <span class="version-mark">New in v7.4.0</span>
```
Comment (ran-huang) [inline system-variables.md:2264]: please sort system variables in alphabetical order

**Review by ran-huang — APPROVED** [review]

**Pull request closed without merging**